### ISSUE TITLE
Jodit 3.4.2 fails in Webpack project

### ISSUE THREAD

**Issue by keithrz** [xdan/jodit#414]
**Jodit Version:**  3.4.2

**Browser:**  Chrome
**OS:**  Mac

**Code / Steps to recreate**

0. If you don't have create-react-app, install it via npm i -g create-react-app
1. create-react-app jodit-test-react
(this will create a new npm project in subfolder jodit-test-react)
2. In package.json, add the following to dependencies:
    "jodit": "3.3.24”,
3. npm i
4. In src/App.js - add textarea, add imports, add effect that calls Jodit.make
```js
import React, { useEffect } from 'react';
import logo from './logo.svg';
import './App.css';
import Jodit from 'jodit';
import 'jodit/build/jodit.min.css';

function App() {
  useEffect(() => {
    Jodit.make('#editor');
  }, []);

  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <textarea id="editor" name="editor"></textarea>
      </header>
    </div>
  );
}

export default App;
```
5. npm start
6. In browser, navigate to http://localhost:3000, and verify jodit works as expected with this version (3.3.24)
7. stop npm start process
8. Modify package.json - change jodit version from 3.3.24 to 3.4.2
9. npm i
10. npm start
11. In browser, navigate to http://localhost:3000.  You'll now get the following error:
TypeError: Cannot read property 'make' of undefined

**Expected behavior:**
On step 11, using version 3.4.2, no error occurs.  Jodit is rendered in place of the textarea.

**Actual behavior:**
error from calling Jodit.make:
TypeError: Cannot read property 'make' of undefined

Note: I do not get the error if I use build/jodit.min.js directly in an index.html file.  

I think i would get the error in any project where I tried importing it in this way, in a project that is using webpack.  I used create-react-app only because it uses webpack out-of-the-box, and so the steps to recreate were straightforward (at least for me :) )  Perhaps the issue stems from the fact that my test project is not using TypeScript, only JavaScript?

Thanks for the great library!!!

**Comment by rockwood** [comment]
I'm using typescript and webpack. I just tried updating from v3.3 and ran into the same issue.

**Comment by keithrz** [comment]
If I change the import to this:

`import { Jodit } from 'jodit';`

Jodit is at least something other than undefined.  With this modified import, I can call Jodit.make() or new Jodit() without getting an error like "cannot call 'make' on undefined", but I still get an error:

```
Uncaught ReferenceError: Jodit is not defined
    at Object.t.isJoditObject (jodit.min.js:9)
    at Object.t.makeCollection (jodit.min.js:9)
    at Jodit.t [as constructor] (jodit.min.js:9)
    at new Jodit (jodit.min.js:9)
    at Function.Jodit.make (jodit.min.js:9)
    at App.js:9

```

@rockwood, if you try this modified import in a TypeScript app, do you get the same error?

@xdan please let me know if you need more information.

**Comment by rockwood** [comment]
@keithrz I also get that error.

**Comment by keithrz** [comment]
It looks like the issue is fixed in 3.4.6.  With 3.4.6 or later, it works if I change the import to use named export instead of default export, that is:

`import { Jodit } from 'jodit';`

before 3.4, the import I used was:

`import Jodit from 'jodit';`
